pennylane/ops/__init__.py

```diff
@@ -54,6 +54,10 @@ class Identity(CVObservable):
     ev_order = 1
     eigvals = np.array([1, 1])
 
+    @classmethod
+    def _eigvals(cls, *params):
+        return cls.eigvals
```

Comment:
Out of curiosity, is this needed? I thought the `eigvals` defined on line 55 would overwrite the `Observable.eigvals` property.

Reply:
It does. But we don't want to duplicate the code and reuse `eigvals` to define `_eigvals`

Reply:
I think that's the bit I'm missing, why does `_eigvals` need to be defined here?

Reply:
It's not used technically, but if we don't define it like this it will automatically fall back to `np.linalg.eigvals(_matrix)` which is overdoing it considering that we already hardcoded the eigenvalues